backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatusTransitions.sql

```diff
@@ -59,4 +59,13 @@ INSERT INTO dbo.[ProjectStatusTransitions] (
 ), (
     11 -- Approved for ERP
     , 14 -- Cancelled
+), (
+    11 -- Approved for ERP
+    , 15 -- On Hold
+), (
+    15 -- On Hold
+    , 12 -- On Hold
```

Comment:
Shouldn't it be `12 -- Approved for SPL` ? 
Same with the rest below?

Reply:
Your comments don't line up